doc/project/checklist.md

```diff
@@ -75,11 +75,13 @@ Block diagrams have been updated to reflect the current design.
 
 All IP block interfaces that are not autogenerated are documented.
 
-### DOC_INTEGRATION_GUIDE
+### DOC_INTEGRATION_D2
 
 Any integration specifics that are not captured by the [comportability]({{< relref "doc/rm/comportability_specification" >}}) specification have been documented.
 Examples include special synthesis constraints or clock requirements for this IP.
 
+In case the IP supports multiple parameterizations (e.g. instantiation parameters or memory maps), the configurations that are planned to be verified and signed off as part of the V* stages must be documented in a section called "Supported Parameter Sets".
```

Comment:
Re. `or memory maps`

I think you are referring to `otp_ctrl` which refers to parameters and types from the generated `otp_ctrl_part_pkg.sv`, which goes along the same lines as other top-specific blocks that also reference parameters and types from generated packages (*mgr, alert_handler etc). The design configurations for these live in separate scopes (i.e. in separate top level specific areas), so IMO they are not different configurations of the same IP - they are totally separate IPs that need to be independently signed off. It will be great to follow a uniform methodology for all such IPs (such as ipgen) to reduce ambiguity (hopefully we will get there soon!). 

The problem here I think, is limited to the in-RTL design parameterizations, where all supported parameter sets that go into the same top level need to be signed off independently. Unless all those supported parameters sets are properly documented, it is not obvious what configurations to test, given it's all in the same scope. 

So maybe, it is okay to omit `or memory maps` from here. Let me know if I have misunderstood something here.